tests/lightning_qubit/test_measurements_class.py

```diff
@@ -584,6 +591,10 @@ def test_double_return_value(self, shots, measurement, obs0_, obs1_, lightning_s
             [[1, 0], [0, 1]],
         ],
     )
+    @pytest.mark.skipif(
```

Comment:
Does that fail?

Reply:
It should not fail but to add this functionality I need to add several functions in `lightning_kokkos.py. Although, I want to keep this PR as simple as possible to make it easier for review but, for sure, this functionality will be added in the future. :smile: